tests/parser/ref-followed-by-eof-crash.error

```diff
@@ -0,0 +1,5 @@
+Error: expected enum keyword
+----
+ 0 | ref
+     ^- expected enum keyword
```

Comment:
This actually is not correct, the highlight should be after the `ref` word, unfortunately using the next token's span causes error message not to show up:
```
Error: expected enum keyword
----
 0 | ref_
----

```
Seems like a bug in error reporting logic